internal/pkg/db/db_test.go

```diff
@@ -121,6 +122,8 @@ func consolidateEdges(edges []*sbom.Edge) []*sbom.Edge {
 
 	for typedEdge, toIDs := range edgeMap {
 		if len(toIDs) > 0 {
+			sort.Strings(toIDs)
```

Comment:
I made a similar change in the same line, so one of us is going to get a conflict. I used `slices` because it was already imported

```suggestion
			slices.Sort(toIDs)
```